tests/TestExternalReactions.py

```diff
@@ -23,6 +23,12 @@
 
 from JSBSim_utils import JSBSimTestCase, CreateFDM, RunTest, CopyAircraftDef
 
+def getParachuteArea(tree):
+    parachute_area = 1.0
+    for value in tree.getroot().findall('external_reactions/force/function/product/value'):
+        parachute_area *= float(value.text)
+    return parachute_area
+
```

Comment:
Ah, I see with the `findall` you're effectively picking up both the parachute area and also the drag coefficient. 

https://github.com/JSBSim-Team/jsbsim/blob/c9b12c4794046ae1406e1a66941937a64417bfd5/aircraft/ball/ball.xml#L104-L112

Reply:
@bcoconni - it all looks good to me!

Reply:
> Ah, I see with the `findall` you're effectively picking up both the parachute area and also the drag coefficient.

Yep because when I've been bitten once, I try to avoid being bitten a second time 😉

Reply:
Yep, was just initially confusing to me since the function name, variable name etc. referred to parachute area but I didn't see anything specific to parachute area in the query, so was wondering how it worked.